test/unit/spec/LoginRouter_spec.js

```diff
@@ -975,6 +975,48 @@ function (Okta, Q, Backbone, SharedUtil, CryptoUtil, CookieUtil, Logger, OktaAut
           expect(test.form.selectedCountry()).toBe('Nihon');
         });
       });
+
+      itp('overrides text for keys in courage bundle for non english languages', function () {
+        return setupLanguage({
+          settings: {
+            language: 'NL',
+            i18n: {
+              'nl': {
+                'oform.errorbanner.title': 'Dutch error banner title'
+              }
+            }
+          }
+        })
+        .then(function (test) {
+          test.form.submit();
+          expect(test.form.errorMessage()).toBe('Dutch error banner title');
+        });
+      });
+
+      itp('Text for keys in courage bundle are in jp as set in settings.language', function () {
+        return setupLanguage({
+          mockLanguageRequest: 'ja',
+          settings: {
+            language: 'ja'
+          }
+        })
+        .then(function(test){
+          test.form.submit();
+          expect(test.form.errorMessage()).toBe('Japanese error banner title');
+        });
+      });
+
+      itp('Text for keys in courage bundle are in en as set in settings.language', function () {
```

Comment:
This might be make more sense as ```Strings in courage bundle are in English by default``` and remove the ```settings``` object. Thinking because we already tested that a specific language works in the test above (jp)